src/axom/mir/docs/sphinx/mir_algorithms.rst

```diff
@@ -0,0 +1,153 @@
+| ``matset: name``                | A required string argument that specifies the name   |
+|                                 | of the matset that will be operated on.              |
++---------------------------------+------------------------------------------------------+
+| ``matsetName: name``            | An optional string argument that specifies the name  |
+|                                 | of the matset to create in the output. If the name   |
+|                                 | is not given, the output matset will have the same   |
+|                                 | name as the input matset.                            |
++---------------------------------+------------------------------------------------------+
+| ``originalElementsField: name`` | The name of the field in which to store the original |
+|                                 | elements map.                                        |
++---------------------------------+------------------------------------------------------+
+| ``selectedZones: [zone list]``  | An optional argument that provides a list of zone ids|
+|                                 | on which to operate. The output mesh will only have  |
+|                                 | contributions from zone numbers in this list, if it  |
+|                                 | is given.                                            |
++---------------------------------+------------------------------------------------------+
+| ``topologyName: name``          | The name of the new topology in the output mesh. If  |
+|                                 | it is not provided, the output topology will have the|
+|                                 | same name as the input topology.                     |
++---------------------------------+------------------------------------------------------+
+
+###############
+EquiZAlgorithm
+###############
+
+The `Equi-Z MIR algorithm <https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.osti.gov/servlets/purl/15014510&ved=2ahUKEwittMui-euIAxUzxOYEHXTWA2kQFnoECBcQAQ&usg=AOvVaw3qbX9qgwCn4qDP0iZ3Sq0J>`_ by J. Meredith 
+is a useful visualization-oriented algorithm for MIR. Whereas many MIR algorithms 
+produce disjointed element output, Equi-Z creates output that mostly forms continuous
+surfaces and shapes. Continuity is achieved by averaging material volume fractions to
+the mesh nodes for each material and then performing successive clipping for each
+material, using the node-averaged volume fractions to determine where clipping occurs
+along each edge. The basic algorithm is lookup-based so shape decomposition for a
+clipped-zone can be easily determined. The clipping stage produces numerous zone fragments
+that are marked with the appropriate material number and moved into the next material
+clipping stage. This concludes when all zones are comprised of only 1 material. From,
+there points are made unique and a new output mesh is created.
```

Comment:
From the discussion and the concentric rings example, I take it that Equi-Z can handle meshes with more than two materials.  Is this correct?

Can Equi-Z handle mixed zones containing more than two materials?

If handling meshes and zones with more than two materials is pretty normal for MIR algorithms, great; otherwise, I think you should state it in this paragraph.